src/Raven.Server/RavenServer.cs

```diff
@@ -441,6 +481,22 @@ private void StartMonitoringCpuCredits()
 
                 try
                 {
+                    if (i++ == (int)Configuration.Server.CpuCreditsExecSyncInterval.AsTimeSpan.TotalSeconds)
```

Comment:
Use `Stopwatch` and check for time, rather than assume that the `Delay` would be seconds

Reply:
I assumed since this is all heuristic-based, it doesn't matter if it's going to be 30 minutes or 32 minutes... Does the accuracy matter here?

Reply:
I don't like it because it isn't clear what you are doing.
The accuracy doesn't matter, but to understand the code you have to jump around a bit, and it is better if we said what we meant, rather than inferred it.

Reply:
ok